app/src/main/java/com/framgia/wsm/screen/login/LoginViewModel.java

```diff
@@ -1,16 +1,45 @@
 package com.framgia.wsm.screen.login;
 
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import com.android.databinding.library.baseAdapters.BR;
+import com.framgia.wsm.R;
+import com.framgia.wsm.data.source.remote.api.error.BaseException;
+import com.framgia.wsm.utils.navigator.Navigator;
+import com.framgia.wsm.utils.validator.Rule;
+import com.framgia.wsm.utils.validator.ValidType;
+import com.framgia.wsm.utils.validator.Validation;
+import com.framgia.wsm.utils.validator.Validator;
```

Comment:
ERROR: Unused import: com.framgia.wsm.utils.validator.Validator. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports